### PR TITLE
Add `communicate-on-pull-request-released` GitHub Action

### PR DESCRIPTION
**Pull request by mollyIV**
### Description

The action will:

* add a comment to the pull request about including the changes in a release
```
Congratulations! 🎉 This was released as part of fastlane x.y.z 🚀
```
* add `status: released` label to the pull request and remove `status: included-in-next-release` label from it if existing
* add a comment to the referenced issue that the pull request has been released
```
The pull request #xxxxx that closed this issue was merged and released as part of fastlane x.y.z 🚀
Please let us know if the functionality works as expected as a reply here. If it does not, please open a new issue. Thanks!
```

Please note that this is just an action implementation. It does not take any effect itself. A new workflow which uses the action will be created.

### Testing

**Manual testing**

A workflow file used for a manual testing:

```yml
name: Processing release
on: 
  release:
    types: [published]

jobs:   
  communicate-on-pull-request-released:
    name: PR released
    runs-on: ubuntu-latest    
    steps:
    - uses: actions/checkout@master
      with:
        ref: refs/heads/master
    - uses: mollyIV/github-actions/communicate-on-pull-request-released@communicate-on-pull-request-released
      with:
        repo-token: "${{ secrets.BOT_TOKEN }}"
```

Please not that the action trigger is a [`release [published]`](https://help.github.com/en/github/automating-your-workflow-with-github-actions/events-that-trigger-workflows#release-event-release).

My bot is set to my private account, that's why you see the comments from the author.

image

A comment added to pull requests number 4 and 6. 

image

A comment added to a referenced issue number 1 from a pull request description's.

image

🎉 

**Unit testing**

To test the changes, run unit tests by executing npm test (followed by npm install to install the package dependencies).

### CODE REVIEW

File: communicate-on-pull-request-merged/__tests__/main.test.ts [hunk 1]
```diff
@@ -49,32 +49,32 @@ describe('action test suite', () => {
...
-      await main.run();
+  //     const main = require('../src/main');
+  //     await main.run();
 
-      expect(api.isDone()).not.toBeTruthy();
-    });
-  }
+  //     expect(api.isDone()).not.toBeTruthy();
+  //   });
+  // }
```
Comment (janpio) [inline communicate-on-pull-request-merged/__tests__/main.test.ts]: you probably want to remove this code

File: communicate-on-pull-request-released/__tests__/main.test.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      process.env['GITHUB_EVENT_PATH'] = path.join(
+        __dirname,
+        scenario.response
+      );
+
+      const api = nock('https://api.github.com')
+        .persist()
+        .post(
+          '/repos/foo/bar/pulls/999/reviews',
+          '{"body":"Congratulations! :tada: This was released as part of [_fastlane_ 2.134.1](https://github.com/Codertocat/Hello-World/runs/128620228) :rocket:","event":"COMMENT"}'
```
Comment (janpio) [inline communicate-on-pull-request-released/__tests__/main.test.ts]: Are we currently using `_fastlane_` (vs. `fastlane`)?

File: communicate-on-pull-request-merged/__tests__/main.test.ts (hunk 1 above)
Comment (mollyIV) [reply communicate-on-pull-request-merged/__tests__/main.test.ts]: Nice catch 💪

File: communicate-on-pull-request-released/__tests__/main.test.ts (hunk 2 above)
Comment (mollyIV) [reply communicate-on-pull-request-released/__tests__/main.test.ts]: Yes, we are 

https://github.com/fastlane/issue-bot/blob/457348717d99e5ffde34ca1619e7253ed51ec172/bot.rb#L271

**Pull request merged**